content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,101 @@
+===============================================================
+Meetup 2023 September 20th in NYCU (do-unknown-things training)
+===============================================================
+
+:date: 2023-08-31 13:00
+:url: meetup/2023/0920-nycu
+:save_as: meetup/2023/0920-nycu.html
+
+
+Date
+-----
+
+* Date: September 20th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:00 Checking in
+* 19:00-20:30 Training
```

Comment:
There should still be a time table for the training.  @EN-Chou you may add it under the training section.

Reply:
Hi @yungyuc, I've added the time table in the training section with a title "Time table for the training" in bold letters.